### ISSUE TITLE
IllegalStateException after closing player

### ISSUE THREAD

**Issue by SergioPoq** [bambuser/BambuserPlayerSDK-Android#10]
After using the seekbar and closing the player, sometimes the app crashes with the following error:

```
java.lang.IllegalStateException
	at android.media.MediaPlayer.getCurrentPosition(Native Method)
	at com.bambuser.broadcaster.NiceMediaPlayer.getCurrentPosition(SourceFile:2)
	at com.bambuser.broadcaster.BroadcastPlayer.getCurrentPosition(SourceFile:2)
	at com.bambuser.player.m5.d(SourceFile:126)
	at com.bambuser.player.yd.run(SourceFile:4)
	at java.util.TimerThread.mainLoop(Timer.java:562)
	at java.util.TimerThread.run(Timer.java:512)
```

**Comment by JennyBambuser** [comment]
Thank you for your feedback, we are currently looking into the issue

**Comment by SimovicIvan** [comment]
Should be fixed in 1.1.5, please let us know if you can still reproduce.
